msvc/libmonoutils.vcxproj

```diff
@@ -42,6 +42,9 @@
     <ClCompile Include="..\mono\utils\mono-io-portability.c" />
     <ClCompile Include="..\mono\utils\mono-linked-list-set.c" />
     <ClCompile Include="..\mono\utils\mono-logger.c" />
+    <ClCompile Include="..\mono\utils\mono-log-windows.c" />
+    <ClCompile Include="..\mono\utils\mono-log-common.c" />
+    <ClCompile Include="..\mono\utils\mono-log-posix.c" />

```

Comment:
Removed.

From: Johan Lorensson +ADw-notifications+AEA-github.com+ADw-mailto:notifications+AEA-github.com+AD4APg-
Reply-To: mono/mono +ADw-reply++0004b6faa950165f737b9b10814f48cc21f476afba774d8192cf000000011369093992a170ce03ea24AQA-reply.github.com+ADw-mailto:reply++0004b6faa950165f737b9b10814f48cc21f476afba774d8192cf000000011369093992a170ce03ea24AQA-reply.github.com+AD4APg-
Date: Friday, June 3, 2016 at 5:00 AM
To: mono/mono +ADw-mono+AEA-noreply.github.com+ADw-mailto:mono+AEA-noreply.github.com+AD4APg-
Cc: Neale Ferguson +ADw-neale+AEA-sinenomine.net+ADw-mailto:neale+AEA-sinenomine.net+AD4APg-, Mention +ADw-mention+AEA-noreply.github.com+ADw-mailto:mention+AEA-noreply.github.com+AD4APg-
Subject: Re: +AFs-mono/mono+AF0- Enhance log tracing (+ACM-2734)


In msvc/libmonoutils.vcxproj+ADw-https://github.com/mono/mono/pull/2734+ACM-discussion+AF8-r65676619+AD4-:

+AD4- +AEAAQA- -42,6 ++40-,9 +AEAAQA-
+AD4-      +ADw-ClCompile Include+AD0AIg-..+AFw-mono+AFw-utils+AFw-mono-io-portability.c+ACI- /+AD4-
+AD4-      +ADw-ClCompile Include+AD0AIg-..+AFw-mono+AFw-utils+AFw-mono-linked-list-set.c+ACI- /+AD4-
+AD4-      +ADw-ClCompile Include+AD0AIg-..+AFw-mono+AFw-utils+AFw-mono-logger.c+ACI- /+AD4-
+AD4-     +ADw-ClCompile Include+AD0AIg-..+AFw-mono+AFw-utils+AFw-mono-log-windows.c+ACI- /+AD4-
+AD4-     +ADw-ClCompile Include+AD0AIg-..+AFw-mono+AFw-utils+AFw-mono-log-common.c+ACI- /+AD4-
+AD4-     +ADw-ClCompile Include+AD0AIg-..+AFw-mono+AFw-utils+AFw-mono-log-posix.c+ACI- /+AD4-


mono-log-posix.c won't be used on windows so if its added into vcxproj file it should be excluded from build or it could generate empty object file triggering unnecessary build warnings.

+IBQ-
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub+ADw-https://github.com/mono/mono/pull/2734/files/583d1a2580a5fc61d0002920c17c23fa71375c3d+ACM-r65676619+AD4-, or mute the thread+ADw-https://github.com/notifications/unsubscribe/AAS2-mx5cCweLD7Gl0F74pU7KEMeo86oks5qH-05gaJpZM4HsGtY+AD4-.